src/components/organisms/Header/Header.tsx

```diff
@@ -7,9 +7,9 @@ export interface HeaderProps {
 
 export const Header = ({ leftButton, breadcrumbs }: HeaderProps) => {
   return (
-    <Grid justify="space-between">
-      <Col span="auto">{breadcrumbs}</Col>
-      <Col span="content">{leftButton}</Col>
-    </Grid>
+    <Group position="apart">
```

Comment:
I changed this [here](https://github.com/fullstacksjs/the-charity-frontend/blob/722fbdb41edcfaa5894b3de41bbadf24fd01802f/src/components/organisms/Header/Header.tsx)